frontend/src/pages/cohorts/CohortsPage.tsx

```diff
@@ -6,8 +6,8 @@ import { Dispatch, SetStateAction, useState } from "react";
 import {
   MAX_ROWS_EXPORT,
   MAX_ROWS_EXPORT_WARNING,
-  WesSampleColumns,
-  CohortColumns,
+  wesSampleColDefs,
+  cohortColDefs,
```

Comment:
Renaming these to make clearer that they're column definitions and to follow JS practices of reserving pascal case for classes, camel case for regular variables.